genienlp/arguments.py

```diff
@@ -564,7 +564,7 @@ def parse_argv(parser):
         '--e2e_dialogue_valid_submetrics',
         nargs='+',
         type=str,
-        default=['jga', 'em', 'em'],
+        default=['dst_em', 'em', 'da_em'],
```

Comment:
To confirm I understand this correctly, `em` is actual exact match, `dst_em` and `da_em` normalize first, then do exact match, right? Can you mention this in a comment?

Reply:
Yeah